app/views/filters/index.html.erb

```diff
@@ -1,3 +1,48 @@
 <%= turbo_frame_tag 'frame_filters' do %>
-  <p>You have a total of <%= keywords_count %> keywords.</p>
+  <p>
+    <%= t('filters.total_of_keywords', keywords_count: keywords_count) %>
+  </p>
+  <%= form_with url: keywords_path, method: :get, class: 'form-filter', data: { turbo: true, turbo_frame: 'frame_list_keywords' } do |form| %>
+    <div>
+      <%= form.label :keyword_pattern, class: 'form-label' do %>
+        <i class="bi bi-list-check"></i> <%= t('filters.by_keyword') %>...
+      <% end %>
+      <%= form.text_field :keyword_pattern, placeholder: 'type a keyword pattern', class: 'form-control' %>
+    </div>
+    <div>
+      <%= form.label :url_pattern, class: 'form-label mt-3' do %>
+        <i class="bi bi-link-45deg"></i>
+        <%= t('filters.result_link') %>...
+      <% end %>
+      <%= form.text_field :url_pattern, placeholder: 'type a url pattern', class: 'form-control' %>
+    </div>
+    <div class="mt-3">
+      <%= form.collection_check_boxes :link_types, ResultLink.link_types, :last, :first do |check_box| %>
+        <%= check_box.check_box %>
+        <%= check_box.label do %>
+          <%= t("link_types.#{check_box.text}") %>
+        <% end %>
+      <% end %>
+    </div>
+    <div class="d-flex justify-content-end">
+      <%= form.button t('filters.submit_btn'), class: 'btn btn-outline-primary mt-3' %>
+    </div>
+  <% end %>
+  <div class="form-text mt-3">
+    <i class="bi bi-info-square"></i>
+    <%= t('filters.cheatsheet') %>:
+    <ul class="list-group">
+      <li class="list-group-item list-group-item-light"><code>(a|b)</code> 'a' or 'b'</li>
```

Comment:
`light` is a modifier here, so it should be `--light` instead `-light`
```suggestion
      <li class="list-group-item list-group-item--light"><code>(a|b)</code> 'a' or 'b'</li>
```

![Screen Shot 2021-08-03 at 11 24 13 AM](https://user-images.githubusercontent.com/6076762/127957604-c3445da1-7c55-48ed-b45b-6d6773dfbe58.png)

Reply:
Hehe, yes well stopped @junan -- it catched my eyes too~
But Bootstrap does not want to follow our conventions :D
https://getbootstrap.com/docs/5.0/components/list-group/#contextual-classes

As this is not a custom class but a vendor class, I think it is OK to let it that way. ;-)

Reply:
To resolve this issue, we usually use `scss`'s `@extend` and make it like BEM

```scss
.list-group-item {
    &--light {
      @extend .list-group-item-light
  }
}
```

https://github.com/junan/google-scraper/blob/develop/assets/stylesheets/base/_pagination.scss

For Web IC's I think it ok to with existing ones